ResearchKit/Common/ORKFormStep.h

```diff
@@ -105,6 +105,21 @@ ORK_CLASS_AVAILABLE
 @interface ORKFormItem : NSObject <NSSecureCoding, NSCopying>
 
 /**
+ Returns an initialized form item using the specified identifier, title, optionality and answer format.
+ 
+ @param identifier    The string that identifies the form item, which should be unique within the form step.
+ @param text          The text displayed as a prompt for the form item's question.
+ @param optional      Is this item optional?
+ @param answerFormat  The answer format for the form item.
+ 
+ @return An initialized form item.
+ */
+- (instancetype)initWithIdentifier:(NSString *)identifier
+                              text:(nullable NSString *)text
+                          optional:(BOOL) optional
+                      answerFormat:(nullable ORKAnswerFormat *)answerFormat;
+
```

Comment:
Lets move optional to the end of the init to stay consistent with the init we already have, which follows the pattern `identifier, text, answer format`.